src/imagery/i.eodag/i.eodag.py

```diff
@@ -37,66 +35,83 @@
 # % answer: S1_SAR_GRD
 # % guisection: Filter
 # %end
-
+# %option G_OPT_V_MAP
+# % description: If not given then current computational extent is used
+# % label: Name of input vector map to define Area of Interest (AOI)
+# % required: no
+# % guisection: Region
+# %end
+# %option
+# % key: clouds
+# % type: integer
+# % description: Maximum cloud cover percentage for Sentinel scene
+# % required: no
+# % guisection: Filter
```

Comment:
I guess this has a default value no?

Reply:
I think not giving it a default value, might be more safe.. Because we don't know if all providers accept cloud cover... So leaving it to be "None" can then be used as an indication to not pass it at all when as a search parameter.

Reply:
Right, then we can add a default value once we use i.eodag within i.landsat or i.sentinel, sometimes I forget that part. I leave this here for reference: https://eodag.readthedocs.io/en/stable/notebooks/api_user_guide/4_search.html?highlight=cloud#Extra-query-parameters